satellite/marketing/marketingweb/server_test.go

```diff
@@ -76,3 +78,46 @@ func TestCreateOffer(t *testing.T) {
 		}
 	})
 }
+
+func TestStopOffer(t *testing.T) {
+	testplanet.Run(t, testplanet.Config{
```

Comment:
We don't need to run another `testplanet` instance here. We can just add these test cases into the first one